Dockerfile

```diff
@@ -14,10 +14,15 @@ RUN npm run build-scss
 
 # Compile Tailwind CSS files
 RUN npx tailwindcss build assets/stylesheets/vendors/tailwind.css -o ./static/css/tailwind.css
+RUN npx tailwindcss build ./static/css/application.css -o ./static/css/application.css
 
 # Minify Javascript files
 RUN npm run minify-js
 
+# Generate SVG Sprite
+RUN npm install svg-sprite -g
+RUN svg-sprite -cD static -cscss assets/images/icons/*.svg
```

Comment:
Why not using the makefile command `assets/icon-sprite` or the NPM script `npm run generate-svg-sprite`?

Reply:
@olivierobert updated on [412cc5d](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/412cc5d70f3d9bf049595eb32b8c7bef12acd357) to use make command